corehq/apps/data_cleaning/templates/data_cleaning/modals/base_confirm.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n %}
+
+<div
+  id="{{ modal_id }}"
+  class="modal fade"
+  tabindex="-1"
+  aria-labelledby="{{ modal_id }}-label"
+  aria-hidden="true"
+  data-bs-backdrop="static"
```

Comment:
Any particular reason this is static? I usually think of static as being for modals that want to force the user to make a decision.

Reply:
if I don't put this in, the modal can be dismissed but the backdrop remains, rendering the whole page unusable until a refresh happens. I tried upgrading the point version of bootstrap locally to see if there was a bugfix, but the same issue remained unless i specified `data-bs-backdrop="static"`. I want to investigate this further later, but right now diving into that rabbit hole any further wouldn't be an appropriate use of my time.

Reply:
that being said, i think we should upgrade the point version of bootstrap 5 anyway, but i will leave that for a dependacat rotation as I want to investigate possible side effects thoroughly...and there isn't time for that right now.

Reply:
Weird. Makes sense.